Documentation/Signoffs/Sign.md

```diff
@@ -0,0 +1,96 @@
+# Sign System Detail Design
+
+
+
+## Function of the Subsystem
+
+The sign that will be implemented into this project will show the current number of available parking spots for the monitored lot.This will be updated roughly as often as the mobile application. The data being sent to the display will be from the primary and secondary data acquisition systems. That data will then be sent to the server and programmed into readable data to the sign. The sign will then consist of a text message stating “Open Spots Available”.Then, there will be two large weatherproof seven segment displays stating how many available spots there are. This will then be controlled by an Arduino that is wirelessly receiving data from the server.This whole system will then need to be waterproofed and mounted to a standing sign.
+
+## Constraints
```

Comment:
Some of these are too ambiguous to be design targets.

Reply:
This hasn't been addressed

Reply:
change this to a list of constraints with explanation only added where necessary.